model/project_ref.go

```diff
@@ -438,6 +441,12 @@ func (p *ProjectRef) MergeWithProjectConfig(version string) error {
 		if projectConfig.TaskSync != nil {
 			pRefToMerge.TaskSync = *projectConfig.TaskSync
 		}
+		if projectConfig.GithubTriggerAliases != nil {
```

Comment:
These don't need if statements since you aren't doing nil pointer references; you can just add to the original definition at 422.